fabtests/common/hmem_cuda.c

```diff
@@ -366,4 +476,10 @@ int ft_cuda_copy_from_hmem(uint64_t device, void *dst, const void *src,
 	return -FI_ENOSYS;
 }
 
+int ft_cuda_get_dmabuf_fd_offset(uint64_t device, void *buf, size_t len,
```

Comment:
There is no `_offset` in the function name.

Reply:
I thought I fixed the names to be consistent with the main, but there are more, I will fix it. Thanks